### PR TITLE
Add Bash class to composer command

### PR DESCRIPTION
**Pull request by franzholz**
Change the style.

### CODE REVIEW

**Review by froemken — APPROVED** [review]

**Comment by franzholz** [comment]
Question:

Shouldn't this be changed into Code-Blocks in order to have a copy button on the right side?

image

**Comment by froemken** [comment]
Currently, I make review only. The change is OK for me. I'm new to docs team...so I don't know all the tiny differences. That's why I have not merged it.

File: Documentation/Installation/DeployTYPO3.rst [hunk 1]
```diff
@@ -21,20 +21,20 @@ General Deployment Steps
...
-    The `composer install` command should not be run on the live environment.
-    Ideally, `composer` should only run locally or on a dedicated deployment machine,
+    The :bash:`composer install` command should not be run on the live environment.
+    Ideally, :bash:`composer` should only run locally or on a dedicated deployment machine,
     to allow testing before going live.
 
     To avoid conflicts between the local and the server's PHP version,
     the server's PHP version can be defined in the :file:`composer.json` file
-    (e.g. ``{"platform": {"php": "7.4.10"}}``), so `composer` will always check
+    (e.g. ``{"platform": {"php": "7.4.10"}}``), so :bash:`composer` will always check
```
Comment (garvinhicking) [inline Documentation/Installation/DeployTYPO3.rst:37]:
```suggestion
    (e.g. ``{"platform": {"php": "7.4.10"}}``), so Composer will always check
```

I think this isn't related to the command, makes it more readable without highlighting here

File: Documentation/Installation/DeployTYPO3.rst [hunk 2]
```diff
@@ -21,20 +21,20 @@ General Deployment Steps
...
 -  Copy files to the production server
 -  Copy the database to the production server
 -  Clearing the caches
 
 .. note::
 
-    The `composer install` command should not be run on the live environment.
-    Ideally, `composer` should only run locally or on a dedicated deployment machine,
+    The :bash:`composer install` command should not be run on the live environment.
+    Ideally, :bash:`composer` should only run locally or on a dedicated deployment machine,
```
Comment (garvinhicking) [inline Documentation/Installation/DeployTYPO3.rst:32]:
```suggestion
    Ideally, Composer should only run locally or on a dedicated deployment machine,
```

**Review by garvinhicking — APPROVED** [review]

**Comment by garvinhicking** [comment]
@franzholz I think this patch needs rebasing, could you check?

I made two changes where I think the reference was meant for the product name, not the command, do you agree?

**Comment by franzholz** [comment]
I have checked the patch.

image

**Comment by garvinhicking** [comment]
> I have checked the patch.

Great, thanks a lot!

**Pull request merged**